### ISSUE TITLE
Allow passing of classes prop to add to different components

### ISSUE THREAD

**Issue by AndySeymour2904** [andrerpena/react-mde#202]
Would be nice to have a classes object that can be passed to `<ReactMde />` such as;
```
classes: {
    preview: 'my-custom-preview',
    text: 'my-custom-text'
}
```
SImilar to `material-ui` [here](https://material-ui.com/api/input/#css)

This would allow more customisation, currently trying to pass classes in `textAreaProps` will remove the other styling.

Happy to create a PR for this :)

**Comment by andrerpena** [comment]
Hey @AndySeymour2904. I like the idea. I will implement it. Thank you

**Comment by andrerpena** [comment]
Released on 7.8.0
